### PR TITLE
Sending status when pg_notifier marks jobs as Aborted

### PR DESCRIPTION
**Pull request by achettyiitr**
**Fixes** # (*issue*)

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
## Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added unit tests for the code
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: services/pgnotifier/pgnotifier.go [hunk 1]
```diff
@@ -272,10 +274,19 @@ func (notifier *PgNotifierT) updateClaimedEvent(id int64, ch chan ClaimResponseT
 									THEN CAST ( '%[3]s' AS pg_notifier_status_type)
 									ELSE  CAST( '%[4]s' AS pg_notifier_status_type)
 									END), attempt = attempt + 1, updated_at = '%[5]s', error = %[6]s
-									WHERE id = %[7]v`, queueName, maxAttempt, AbortedState, FailedState, GetCurrentSQLTimestamp(), misc.QuoteLiteral(response.Err.Error()), id)
+									WHERE id = %[7]v`, queueName, maxAttempt, AbortedState, FailedState, GetCurrentSQLTimestamp(), misc.QuoteLiteral(response.Err.Error()), claim.ID)
 			_, err = notifier.dbHandle.Exec(stmt)
+
+			// Sending stats when we mark pg_notifier status as aborted.
+			if claim.Attempt > maxAttempt {
```
Comment (dhawal1248) [inline services/pgnotifier/pgnotifier.go:281]: don't we need to increment claim.Attempt somewhere? It is set at claim time and not incremented anywhere else

**Review by dhawal1248 — APPROVED** [review]

**Review by dhawal1248 — CHANGES_REQUESTED** [review]
Marked earlier review as approved by mistake.
We need to increment claim.Attempt whenever a update notif arrives

**Review by sivashanmukh — APPROVED** [review]
LGTM

**Review by dhawal1248 — APPROVED** [review]

File: services/pgnotifier/pgnotifier.go (hunk 1 above)
Comment (achettyiitr) [reply services/pgnotifier/pgnotifier.go:281]: Got resolved.

**Pull request merged**